src/TagLib/Ogg/XiphComment.cs

```diff
@@ -52,12 +52,6 @@ public class XiphComment : TagLib.Tag, IEnumerable<string>
 		/// </summary>
 		private string vendor_id;
 
-		/// <summary>
-		///    Contains the field identifier to use for <see
-		///    cref="Comment" />.
-		/// </summary>
-		private string comment_field = "DESCRIPTION";
-
```

Comment:
Why get rid of this?

Reply:
The compiler complains (warning) that this is not used anywhere. This has anyway been removed in the meantime from base (integration #109) so this change is re-done (can be ignored, same as base).